.gitignore

```diff
@@ -1,24 +1,7 @@
+.npm_cache
 node_modules
 npm-debug.log
 
-# TypeScrit temporary/cache files
-**/.baseDir.ts
-**/.tscache/
-tscommand-*.tmp.txt
-
+# Generated for builds
 **/dist/
-
-# IDEA
-.idea
```

Comment:
I think it make sense to leave this folder in ignore list in case any contributor uses WebStorm